CHANGELOG.md

```diff
@@ -2,6 +2,8 @@
 
 ## Pending
 
+- [\#301](https://github.com/arkworks-rs/algebra/pull/201) (ark-ec) Add `GLVParameters` trait definition.
```

Comment:
should be https://github.com/arkworks-rs/algebra/pull/301

Reply:
Thanks, and fixed.